google/cloud/bigquery/client.py

```diff
@@ -48,11 +48,11 @@
 from google.cloud import exceptions
 from google.cloud.client import ClientWithProject
 
+from google.cloud.bigquery._helpers import _del_sub_prop
 from google.cloud.bigquery._helpers import _get_sub_prop
 from google.cloud.bigquery._helpers import _record_field_to_json
 from google.cloud.bigquery._helpers import _str_or_none
 from google.cloud.bigquery._helpers import _verify_job_config_type
-from google.cloud.bigquery._helpers import _del_sub_prop
```

Comment:
I think this change needs to be reverted too, since `_del_sub_prop` is still used.

Reply:
I have added `_del_sub_prop` before `_get_sub_prop` import to maintain.